src/java-runtime/java-runtime.csproj

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{1D4FC8F1-0DA4-4F38-BE68-11AEBA9A0EA4}</ProjectGuid>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+  </PropertyGroup>
+  <Import Project="..\..\Configuration.props" />
+  <Import Project="..\..\build-tools\scripts\Jar.targets" />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <OutputPath>$(XAInstallPrefix)xbuild\Xamarin\Android</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <OutputPath>$(XAInstallPrefix)xbuild\Xamarin\Android</OutputPath>
+  </PropertyGroup>
+  <Import Project="$(MSBuildBinPath)\Microsoft.Common.targets" />
+  <ItemGroup>
+    <AllRuntimeSource Include="mono/**/*.java" />
```

Comment:
Question: should the `.java` sources be within `src/java-runtime/java/...`, i.e. an intermediate `java` folder?

This would help provide a degree of structure, especially if we add additional Java packages into this lib, e.g. eventual integration of more Java.Interop code: https://github.com/xamarin/java.interop/blob/bd8662998aa3b807bd3abdbc11a006a22c97cfb3/src/Java.Interop/java/com/xamarin/java_interop/internal/JavaProxyThrowable.java

(Note that ^^ is in a `com.xamarin.java_interop.internal` package.)